index.js

```diff
@@ -495,6 +537,10 @@ var processEvent = function(event, context) {
     console.log("processing codepipeline notification");
     slackMessage = handleCodePipeline(event)
   }
+  else if(eventSubscriptionArn.indexOf(config.services.codebuild.match_text) > -1 || eventSnsSubject.indexOf(config.services.codebuild.match_text) > -1 || eventSnsMessage.indexOf(config.services.codebuild.match_text) > -1){
```

Comment:
there seems to be a lot of repetition in all of these if conditions, perhaps they could all be made to access their deeply nested config match_text values a bit more safely as well